docs/reference/portal.dataProduct.json

```diff
@@ -862,7 +862,7 @@
                         },
                         {
                             "name": "generalSettings",
```

Comment:
We will have to update the portal experience. Please let me know when you are ready, so that I can help you with that.